pupyl.py

```diff
@@ -0,0 +1,19 @@
+"""
+Manage command line interface
+"""
+
+from pupyl.cli import PupylCommandLineInterface
+from pupyl.search import PupylImageSearch
+from pupyl.web import interface
+
+
+if __name__ == "__main__":
+    cli = PupylCommandLineInterface()
+    args = cli.parser.parse_args()
+    pupyl = PupylImageSearch()
+
+    pupyl.index(
+        args.data_dir
+    )
```

Comment:
@92amartins , as it's constants, prefer names in uppercase (more: [constants](https://www.python.org/dev/peps/pep-0008/#constants)):

```suggestion
    CLI = PupylCommandLineInterface()
    ARGS = CLI.parser.parse_args()
    PUPYL = PupylImageSearch()

    PUPYL.index(
        ARGS.data_dir
    )
```